### PR TITLE
Federation registry agent role.

### PR DESCRIPTION
**Pull request by Tas-sos**
* [X] Added the Ansible role to deploy the following deployers :
* * [X] [deployer_mitreid.service](https://github.com/rciam/rciam-federation-registry-agent/blob/devel/service/deployer_mitreid.service)
* * [X] [deployer_ssp.service ](https://github.com/rciam/rciam-federation-registry-agent/blob/devel/service/deployer_ssp.service)

With one execution of the role, both are installed. However, the **activation** of each deployer depends on the following variables:
```
enable_ssp: no
enable_mitre: yes
```

### CODE REVIEW

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
* Use `fedreg_agent_` prefix for all vars
* Rciam --> RCIAM

File: roles/fedreg-agent/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+===============================
+
+Federation registry agent (`fedreg-agent`) role deploy and configure the deployers from [rciam-federation-registry-agent](https://github.com/rciam/rciam-federation-registry-agent/tree/devel).
+
+
+Requirements
+------------
+
+* Ansible `2.9.11`.
+* Target to Debian GNU/Linux 10 (buster) hosts.
```
Comment (NicolasLiampotis) [inline roles/fedreg-agent/README.md:11]: Is there anything Debian 10 specific that wouldn't work on 9?

**Comment by NicolasLiampotis** [comment]
Config templates may be affected by changes from https://github.com/rciam/rciam-federation-registry-agent/issues/15

File: roles/fedreg-agent/README.md (hunk 1 above)
Comment (Tas-sos) [reply roles/fedreg-agent/README.md:11]: No, but to be honest, I only tested it in **Debian GNU/Linux 10 (buster)**.

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Could you please rename the playbook from "federation-registry-agent.yml" to "fedregagents.yml". Other than that LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**